nearform/titus
# admin ui via @nearform/brokeneck

Closes https://github.com/nearform/brokeneck/issues/6

This PR adds an admin UI and backend to Titus using brokeneck.

> brokeneck is currently private and its packages are published to Github's package registry

## Testing

- create a GitHub [personal access token](https://github.com/settings/tokens/new) with `write:packages` scope
- authenticate to the registry with `npm login --registry=https://npm.pkg.github.com` (use the generated token above as the password)
- `lerna bootstrap`
- `lerna run create:env`
- configure one of the auth providers (reach out for access information) in .env files of frontend and backend
- set `REACT_APP_ENABLE_ADMIN=true` in frontend .env
- set `ENABLE_ADMIN=true` in backend .env
- `lerna run start:all`
- open `http://localhost:3000`
- authenticate in the UI
- see the "ADMIN" link in the header and click it
- see the users showing up
